automation/jleechanorg_pr_automation/jleechanorg_pr_monitor.py

```
@@ -831,6 +944,660 @@ def _build_codex_comment_body_simple(
+                    model=model,
+                )
+        except Exception as exc:
+            self.logger.error(
+                "❌ Failed to dispatch fix-comment agent for %s #%s: %s",
+                repo_full,
+                pr_number,
+                exc,
+            )
+            return False
+
+    def _process_pr_fix_comment(
```

Ruff (0.14.10)

Too many return statements (8 > 6)

(PLR0911)